pkg/exporter/exporter.go

```diff
@@ -118,6 +119,7 @@ func (e *Exporter) Scrape(ch chan<- prometheus.Metric) {
 				"subject":         cert.Subject.String(),
 				"dns_names":       strings.Join(cert.DNSNames, ","),
 				"email_addresses": strings.Join(cert.EmailAddresses, ","),
+				"hostname":        os.Hostname(),
```

Comment:
`os.Hostname()` returns two values and go vet reports this as an error. See Travis build https://travis-ci.org/amimof/node-cert-exporter/jobs/590563614#L257

I suggest declaring a variable and pass that to `"hostname"`. For example

```go
h, _ := os.Hostname()
```